src/decode/decode_gps_l1ca.c

```diff
@@ -101,18 +102,25 @@ static void decoder_gps_l1ca_process(const decoder_channel_info_t *channel_info,
   if (!subframe_ready(&data->nav_msg))
     return;
 
-  /* Decode ephemeris to temporary struct */
-  ephemeris_t e = {.sid = channel_info->sid};
-  s8 ret = process_subframe(&data->nav_msg, &e);;
-
+  /* Decode nav data to temporary structure */
+  gps_l1ca_decoded_data_t dd;
+  s8 ret = process_subframe(&data->nav_msg, channel_info->sid, &dd);
   if (ret <= 0)
     return;
 
-  /* Decoded a new ephemeris. */
-  ephemeris_new(&e);
 
-  ephemeris_t *eph = ephemeris_get(channel_info->sid);
-  if (!eph->valid) {
-    log_info_sid(channel_info->sid, "ephemeris is invalid");
+  if (dd.gps_l2c_sv_capability_upd_flag) {
+    /* Store new L2C value  */
+    log_info("L2C capabilities received: 0x%x", dd.gps_l2c_sv_capability);
```

Comment:
I vote we remove this log statement since it is irrelevant and confusing for end users on v2.